### ISSUE TITLE
Wheel build fails for backports.zoneinfo 0.2.1 in GitHub Actions: _PyLong_One is undeclared

### ISSUE THREAD

**Issue by alex-kozin** [pganssle/zoneinfo#111]
I am having backports.zoneinfo (0.2.1) package as a dependency of the [dateparser](https://pypi.org/project/dateparser/) package.

Currently, my CI set up involves a docker image of the latest supported ubuntu instance from GitHub Actions. Unfortunately, it fails to build the wheel for zoneinfo. It raises the following error:

```
lib/zoneinfo_module.c: In function ‘zoneinfo_fromutc’:
  lib/zoneinfo_module.c:600:19: error: ‘_PyLong_One’ undeclared (first use in this function); did you mean ‘_PyLong_New’?
    600 |             one = _PyLong_One;
        |                   ^~~~~~~~~~~
        |                   _PyLong_New
  lib/zoneinfo_module.c:600:19: note: each undeclared identifier is reported only once for each function it appears in
  error: command '/usr/bin/gcc' failed with exit code 1
  ----------------------------------------
  ERROR: Failed building wheel for backports.zoneinfo
```

I looked up the [source code](https://github.com/pganssle/zoneinfo/blob/c54d700fcba78b20733390ee3014809ad69858f6/lib/zoneinfo_module.c#L599), but I am not too familiar with CPython and the package itself. Is there a dependency I am missing if the gcc can't build a wheel?

**Comment by pganssle** [comment]
@alex-kozin What version of Python are you building this for?

**Comment by alex-kozin** [comment]
@pganssle This is 3.10, which I just realized differs with the 3.8 I have installed locally. As I understand, 3.10 should not be supported yet? That does solve my problem though, thank you for the question!

**Comment by pganssle** [comment]
I am not sure why it wouldn't install on 3.10, since the code is basically identical to the code in the standard library itself, but in any case `zoneinfo` is now in the standard library, so there's no reason to install `backports.zoneinfo` on Python >= 3.9.

I'm going to close this because I don't intend to make this installable on 3.10 unless we add new features to `zoneinfo` and decide to re-use this package to backport them to earlier Python versions (in which case we'll be adding tests for older Python versions anyway).

**Comment by mjl** [comment]
As an aside, see https://bugs.python.org/issue42161 "Remove private _PyLong_Zero and _PyLong_One variables", which seems to have landed in python 3.10
